src/finch/finch_notation/nodes.py

```diff
@@ -405,9 +425,10 @@ def children(self):
         return [self.name, *self.args, self.body]
 
     @classmethod
-    def from_children(cls, name, *args, body):
+    def from_children(cls, name, *args_body):
```

Comment:
We can't have standard parameter after vararg here.

Reply:
How about changing the order? `body, *args`